tutorials/introductory/customizing.py

```diff
@@ -1,24 +1,40 @@
 """
 .. redirect-from:: /users/customizing
 
+=====================================================
 Customizing Matplotlib with style sheets and rcParams
 =====================================================
 
 Tips for customizing the properties and default styles of Matplotlib.
+There are basically three ways to customize Matplotlib:
+
+    1. :ref:`Using style sheets<customizing-with-style-sheets>`.
+    2. :ref:`Setting rcParams at runtime<customizing-with-dynamic-rc-settings>`.
+    3. :ref:`Changing your matplotlibrc file<customizing-with-matplotlibrc-files>`.
+
+.. _customizing-with-style-sheets:
 
 Using style sheets
-------------------
+==================
 
 The :mod:`.style` package adds support for easy-to-switch plotting
-"styles" with the same parameters as a :ref:`matplotlib rc
+"styles" with the same parameters (rcParams) as a :ref:`matplotlibrc
 <customizing-with-matplotlibrc-files>` file (which is read at startup to
-configure Matplotlib).
+configure Matplotlib). However, in a style sheet you can only set
+rcParams that are related to the actual style of a plot. Other rcParams,
+like *backend*, will be ignored. This behavior is different fom the
+:file:`matplotlibrc` file, which supports all rcParams. In this way,
+style sheets are portable between different machines without having to
+worry about dependencies which might or might not be installed on
+another machine. For a full list of rcParams see `matplotlib.rcParams`.
+For a list of rcParams that are ignored in style sheets see
+`matplotlib.style.use`.
 
 There are a number of pre-defined styles :doc:`provided by Matplotlib
 </gallery/style_sheets/style_sheets_reference>`. For
 example, there's a pre-defined style called "ggplot", which emulates the
-aesthetics of ggplot_ (a popular plotting package for R_). To use this style,
-just add:
+aesthetics of ggplot_ (a popular plotting package for R_). To use this
+style, just add:
```

Comment:
```suggestion
style, add:
```

As long as you are touching this line lets remove "just" which is at best a filler word at worst alienate readers who do not find the following code simple!

Reply:
Done